Data/template.yaml

```diff
@@ -0,0 +1,12 @@
+Material name: ''
```

Comment:
this template does not seem to be correct any more, does it?

Reply:
That is true, indeed!